### ISSUE TITLE
Add team based abs

### ISSUE THREAD

**Issue by vqd8a** [kokkos/kokkos-kernels#209]
Adding abs(team_handle,...)

**Comment by mhoemmen** [comment]
@vqd8a Did you mean to write "dot, axpby, and gemv" instead of "abs" in the title?

**Comment by vqd8a** [comment]
@mhoemmen No, I meant "abs". @crtrott did for dot, axpby, gemv. I will do for other blas functions.

**Comment by mhoemmen** [comment]
@vqd8a I asked that question because the word "abs" does not appear anywhere in the pull request :-) .  The pull request only appears to affect dot, axpby, and gemv.

**Comment by vqd8a** [comment]
@mhoemmen I see your point :-). I don't know why dot, axpby and gemv appear in the PR. Let me try creating a new PR

**Comment by mhoemmen** [comment]
@vqd8a Thanks!

**Comment by mhoemmen** [comment]
@vqd8a Thanks!  Another option is just to rename the PR.
